README.md

```diff
@@ -1,6 +1,8 @@
 # Hello-Books-BE
 
 [![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)
+[![Build Status](https://travis-ci.org/KvNGCzA/Hello-Books-BE.svg?branch=develop)](https://travis-ci.org/KvNGCzA/Hello-Books-BE)
+
 
```

Comment:
Add this to the readme file `[![Maintainability](https://api.codeclimate.com/v1/badges/d75dfd46c0081a6d75b5/maintainability)](https://codeclimate.com/github/KvNGCzA/Hello-Books-BE/maintainability)`

Reply:
ok seen added